storage_service/templates/login.html

```diff
@@ -21,7 +22,7 @@
 </tr>
 </table>
 
-<input type="submit" value="login" class='btn btn-primary' />
+<input type="submit" value="{% trans "login" %}" class='btn btn-primary' />
```

Comment:
Sorry, I was thinking on a `<button>`. That makes sense and it looks like Django doesnt' really care about what value you're passing as long as it is a `POST`.